userguide/tutorials/catalog-examples.adoc

```diff
@@ -0,0 +1,380 @@
+|Trigger invoice dry run with target date `2021-10-17`   |Invoice generated for `$24.95` for `2021-10-17` to `2021-11-17` corresponding to `standard-monthly` plan 
+|Trigger invoice dry run with target date `2022-09-30`   |Invoice generated for `$275` for `2022-09-30` to `2023-09-30` corresponding to `standard-annual` plan  
+|===
+
+As mentioned above, even if there is an addon plan associated with a subscription, the addon plan will be invoiced as per its own date and NOT be aligned with the subscription billing date.
+
+=== Bundle Billing Alignment
+
+When `BUNDLE` billing alignment is specified in the catalog, all subscriptions in a bundle are invoiced together. The bill day is derived from the first bill date of the first subscription. 
+
+This example uses the  https://github.com/killbill/killbill-docs/blob/1a945188c0ef53a3c7f797b4890514d35c09c46f/catalogs/bundle-billing-alignment.xml[Bundle Billing Alignment] catalog. It specifies `BUNDLE` billing alignment. 
+
+
+It has a `BASE` and an `ADD_ON` product as follows:
+
+[options="header",cols="1,1"]
+|===
+|Product   |Category   
+//-------------
+|Standard   |Base   
+|RemoteControl   |Add_on   
+|===
+
+It has two plans corresponding to both products as follows:
+
+[options="header",cols="1,1,1"]
+|===
+|Plan Name   |Product|Plan Description
+//----------------------
+|standard_monthly   |Standard   |Single `EVERGREEN` phase with a monthly recurring charge of `$24.95`   
+|remotecontrol-monthly   |RemoteControl   |Single `EVERGREEN` phase with a monthly recurring charge of `$17.95`    
+|===
+
+The following table lists the actions and their results:
+
+[options="header",cols="1,1"]
+|===
+|Action   |Result   
+//-------------
+|Create a new account. Specify BCD as `25`  |Account created with BCD set to `25`
+|Create a subscription for the `standard-monthly` plan on `2021-09-20`   |Invoice generated for `$24.95` for `2021-09-20` to `2021-10-20` (BCD is ignored, subscription is billed as per its creation date)
+|Trigger invoice dry run with target date `2021-10-20`   |Invoice generated for `$24.95` for `2021-10-20` to `2021-11-20`
+|Add the `remotecontrol-monthly` add-on to the existing subscription with date as `2021-09-30`   |No invoice generated
+|Trigger invoice dry run with target date `2021-09-30`   |Invoice generated for `$11.97` for `2021-09-30` to `2021-10-20` corresponding to the `remotecontrol-monthly` plan (Prorated since `BUNDLE` billing alignment is used) 
+|Trigger invoice dry run with target date `2021-10-20`   |Invoice generated for `$42.90` for `2021-10-20` to `2021-11-20` corresponding to both plans
+|===
+
+Note that `BUNDLE` billing alignment applies only to subscriptions that are part of the same bundle and does not affect subscriptions created in different bundles. So, when multiple subscriptions are created in different bundles, each one is still billed as per its creation date.
```

Comment:
```suggestion
[NOTE]
*Note:* `BUNDLE` billing alignment applies only to subscriptions within a single bundle. This type of billing alignment does not affect subscriptions across different bundles. So, when multiple subscriptions are created in different bundles, each one is still billed as per its creation date.
```

Reply:
I wanted to make this more clear; not sure if I did. I'm not sure what "one" in the last sentence refers to. It could refer to "bundle" or "subscription." 

Also, this note should go at the beginning of the "BUNDLE Billing Alignment" section because we can't assume readers will read to the very end of a section.

Reply:
@FlowWriter, "one" refers to subscriptions.  I had not added the text at the beginning as I did not want to confuse users with too much information at the start. However, the [note] feature is good, I wasn't aware of it, it differentiates the "note" from the main text. So, I've now added the note to the start of the document.

Reply:
I think the last sentence (since it's repeating what's mentioned in the beginning of the section) isn't necessary. I still think this note belongs in the beginning of the section because you can't assume the reader will read all the way to the end of the section.